src/Avalonia.Controls/SplitButton/SplitButton.cs

```diff
@@ -101,6 +108,15 @@ public SplitButton()
             set => SetValue(FlyoutProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets an <see cref="KeyGesture"/> associated with this control
+        /// </summary>
+        public KeyGesture? HotKey
```

Comment:
I wonder if these docs should be inherited from button. Although I didn't inherit any of the other comments which I probably should have.